wbooks/src/app/screens/Login/index.tsx

```diff
@@ -1,16 +1,49 @@
 import React from 'react';
+import { useForm, Controller } from 'react-hook-form';
+import { useNavigation } from '@react-navigation/native';
 import { ImageBackground, Text, TouchableOpacity } from 'react-native';
 import Form from '@screens/Login/components/Form';
 import bcInicio from '@assets/General/bc_inicio.png';
 
 import styles from './styles';
 
 const Login = () => {
+  const { control, handleSubmit, errors } = useForm();
+  const navigation = useNavigation();
+
+  const onSubmit = () => navigation.navigate('Library');
+
   return (
     <ImageBackground source={bcInicio} style={styles.container}>
-      <Form title="Email" />
-      <Form title="Password" isSecure={true} />
-      <TouchableOpacity style={styles.loginBtn} onPress={() => console.log('submit')}>
+      <Controller
+        name="email"
+        control={control}
+        render={({ onChange, value }) => <Form title="Email" value={value} onChange={onChange} />}
+        rules={{
+          required: { value: true, message: 'Campo requerido' },
+          validate: value =>
+            value.match(/^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/) ||
+            'El email no es valido'
```

Comment:
válido

Reply:
also always extract all validations inside a file in `utils` folder where the file should be called `validations.ts`

Reply:
Done!